src/main/java/brave/opentracing/BraveTracer.java

```diff
@@ -56,27 +57,28 @@
    * Returns an implementation of {@linkplain io.opentracing.Tracer} which delegates
    * the the provided Brave Tracer.
    */
-  public static BraveTracer wrap(brave.Tracer brave4) {
+  public static BraveTracer create(Tracing brave4) {
     return newBuilder(brave4).build();
   }
 
-  public static Builder newBuilder(brave.Tracer brave4) {
+  public static Builder newBuilder(Tracing brave4) {
     return new Builder(brave4);
   }
 
   public static final class Builder {
-    brave.Tracer brave4;
+    Tracing brave4;
     Map<Format<TextMap>, Propagation<String>> formatToPropagation = new LinkedHashMap<>();
```

Comment:
This is actually not regarding this pull request, but why limit format to TextMap so that it is not possible to register a io.opentracing.propagation.Format.Builtin#BINARY?